docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: police_data
+    volumes:
+      - ./init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
```

Comment:
I'd use this as the volume `- postgres:/var/lib/postgresql/data`

with this at the end of the file:

```yaml
volumes:
  postgres: {}
```

and I'd also connect this service to the to the dotenv file with:

```yaml
    env_file:
      - ".env"
```

Reply:
I've added both

Reply:
sorry it wasn't clear, also replace

```yaml
    volumes:
      - ./init-user-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
```

with this:

```yaml
    volumes:
      - postgres:/var/lib/postgresql/data
```

and then delete the `init-user-db.sh` file.

This appears to be a more standard way to dockerize Postgres databases that helps to persist data across migrations and deployments.